style/modules/grid.css

```diff
@@ -0,0 +1,20 @@
+.gridContainer{ /* a container with grid */
```

Comment:
we'll want a space between the class name and the left brace, it's just for looks but it's the convention used nearly everywhere